src/addonmanager.cpp

```diff
@@ -3,15 +3,15 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "addonmanager.h"
-#include "constants.h"
 #include "leakdetector.h"
 #include "logger.h"
 #include "models/feature.h"
 #include "qmlengineholder.h"
-#include "signature.h"
 #include "taskscheduler.h"
 #include "tasks/addon/taskaddon.h"
 #include "tasks/function/taskfunction.h"
+#include "addondirectory.h"
```

Comment:
what about if we move all the addon* files into the `addons` folder? This one included.

Reply:
alphabetic order of the includes.